src/Pages/Login/Login.jsx

```diff
@@ -0,0 +1,84 @@
+import { useAuth } from "../../Contexts";
+import { useState, useEffect } from "react";
+import { Navbar, Sidebar } from "../../Components";
+
+import { Link } from "react-router-dom";
+import "./Login.css";
+
+export function Login() {
+  const [username, setUsername] = useState();
+  const [password, setPassword] = useState();
+  const [passwordStyle, setPasswordStyle] = useState({});
+
+  const { currentUser, loginUserWithCredentials, logoutUser } = useAuth();
+
+  function validatePassword(password) {
+    const passwordRegex = /^[\w!@#$%^&*)(+=._-]{6,}$/;
+    return passwordRegex.test(password);
+  }
+
+  useEffect(() => {
+    if (password) {
+      setPasswordStyle(
+        validatePassword(password)
+          ? { backgroundColor: "#8ac926", color: "white" }
+          : { backgroundColor: "#ff595e", color: "white" }
+      );
+    } else {
+      setPasswordStyle({});
+    }
+  }, [password]);
+
+  const loginHandler = () => {
+    loginUserWithCredentials(username, password);
+  };
```

Comment:
hey rohit, just a small suggestion here
you can login user with email and password as credentials because most of the users save their password in the browser and as i faced this issue, google chrome took my email-id as username.
You can check this out!